backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabaseExecutor.java

```diff
@@ -31,8 +30,8 @@ static void executeCreateUser(SqlDatabase db, String user) {
       }
       // session_user has all roles, needed for 'set role'
       jooq.execute("GRANT {0} TO session_user WITH ADMIN OPTION", name(MG_USER_PREFIX + user));
-      // save metadata
-      MetadataUtils.saveUserMetadata(jooq, new User(db, user));
+      // save metadata, don't need database for this operation
+      MetadataUtils.saveUserMetadata(jooq, new User(null, user));
```

Comment:
passing null is usually a bit of a cod smell, why a new user , and if we need a new user maybe a a constructor that does not need a db. i.e .it should be clear db can either be null or not null

Reply:
I considered to create another constructor, but in this case I think the developer using this constructor should carefully consider if 'null' is okay. Alternatively, we could provide a nice exception at the moment the database object is null when requested in 'User'.

Reply:
done